hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/token/CryptoService.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.node.app.service.token;
+
+import com.hedera.node.app.spi.Service;
+import com.hedera.node.app.spi.state.States;
+import javax.annotation.Nonnull;
+
+/**
+ * The {@code CryptoService} is responsible for working with {Account}s. It implements all
```

Comment:
Should `{Account}s` be `{@link Account}s`?

Reply:
Will be changed once the model object `Account` is added in another PR